src/installation/uninstall_epinio.md

```diff
@@ -9,9 +9,11 @@ then removing Epinio will remove those components and this may break your other
 workloads that depended on these. Make sure you understand the implications of
 uninstalling Epinio before you proceed.
 
-If you want to completely uninstall Epinio from your kubernetes cluster, you
-can do this with the command:
+
+Depending on the method you used to install Epinio ([auto](./install_epinio_auto.md) or [manual](./install_epinio_manual.md)),
+the uninstallation method is different. If you used the "auto" method, then you can uninstall
+using the same helm chart:
 
 ```bash
-epinio uninstall
-```
\ No newline at end of file
+$ helm uninstall epinio-installer
+```
```

Comment:
And what do I do with the manual method? It is pretty obvious but we should mention it in one line I think to be consistent.

Reply:
True, I will add something.

Reply:
done